axelrod/strategies/gobymajority.py

```diff
@@ -19,7 +19,7 @@ class GoByMajority(Player):
     default this is 0)
     """
 
-    name = 'Go By Marjority'
+    name = 'Go By Majority'
```

Comment:
Just taken a look: we kind of are catching it. The init method of the strategy renames the instance (which is tested):

```
 53         self.name = (                                                                                                                             
 54             'Go By Majority' + (self.memory > 0) * (": %i" % self.memory))                                                                        
 55         if self.soft:                                                                                                                             
 56             self.name = "Soft " + self.name                                                                                                       
 57         else:                                                                                                                                     
 58             self.name = "Hard " + self.name                                                                                                       
 59                                                  
```

 but yeah we can make that better:

```
 53         self.name = (                                                                                                                             
 54           self.name + (self.memory > 0) * (": %i" % self.memory))                                                                        
 55         if self.soft:                                                                                                                             
 56             self.name = "Soft " + self.name                                                                                                       
 57         else:                                                                                                                                     
 58             self.name = "Hard " + self.name                                                                                                       
 59                                                  
```

I think that would have tripped a test...